src/main/java/io/github/christophermanahan/carnitas/Main.java

```diff
@@ -6,8 +6,8 @@
 public class Main {
 
   public static void main(String[] args) throws IOException {
-    int port = Integer.parseInt(args[0]);
+    int port = args.length == 0  ? 33333 : Integer.parseInt(args[0]);
```

Comment:
👍 

At some point, you might want to look into putting this default into a properties file or something.

Reply:
FYI, this isn't something you need to implement for this PR, maybe something for the future.

Reply:
Got it